dist/index.js

```diff
@@ -1888,6 +1888,17 @@ const runTests = () => {
       cmd.push('--group')
       cmd.push(quoteArgument(group))
     }
+    const configInput = core.getInput('config')
+    if (configInput) {
+      cmd.push('--config')
+      cmd.push(quoteArgument(configInput))
+    }
+    const configFileInput = core.getInput('config-file')
+    if (configFileInput) {
+      cmd.push('--config-file')
+      // TODO should false not be quoted?
```

Comment:
nope, you can confirm that it is passed as string anyway to the test runner from the CLI
```
DEBUG=cypress:cli npx cypress run
DEBUG=cypress:cli npx cypress run --config-file "false"
```